src/error.rs

```diff
@@ -27,6 +27,9 @@ pub enum Error {
     #[error("Could not restart tracee = {pid} with mode = {mode:?}")]
     Restart { pid: Pid, mode: Restart, source: nix::Error },
```

Comment:
We should consider just removing this, since it is just a special case of `TraceeDied`.

Reply:
Will do this in a follow-up PR.